### ISSUE TITLE
Mac builds on release/3.10 are failing

### ISSUE THREAD

**Issue by phargogh** [natcap/invest#773]
We've had a few builds on `release/3.10` failing recently: https://github.com/natcap/invest/commits/release/3.10

and they've all been failing with the message:

```
dist/invest/invest list
qt.qpa.plugin: Could not find the Qt platform plugin "cocoa" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
```

**Comment by phargogh** [comment]
With #774 merged, the builds were at least making it past the binary build step (which is good!) but are now failing on what looks like some secrets not being available to GHA in the code signing step.  I think I've corrected this and am re-running the builds to verify.

**Comment by emlys** [comment]
@phargogh I'm curious which secrets were missing? I could've sworn I updated them all...

**Comment by phargogh** [comment]
`KEYCHAIN_PASS` and `STANFORD_CERT_KEY_PASS` were both undefined in the repo secrets, so I just added them and all seems well

**Comment by emlys** [comment]
Strange, those shouldn't be needed anymore. I removed them in #697.

**Comment by emlys** [comment]
#697 was merged into `main`, which has the correct version of the secrets now. Looks like `release/3.10` somehow didn't get updated accordingly?

**Comment by emlys** [comment]
The build on `release/3.10` passed using the correct code signing setup as recently as yesterday: https://github.com/natcap/invest/runs/4521106822?check_suite_focus=true

**Comment by phargogh** [comment]
Uffda, you're right!  I'm guessing a merge messed things up somewhere along the line

**Comment by phargogh** [comment]
@emlys I'll make a PR to update the Makefile to the way it should be on `release/3.10` and add you as a reviewer so we can get this sorted

**Comment by davemfish** [comment]
I believe the code-signing issues were resolved in #775
